src/Components/dashboard/Dashboard.js

```diff
@@ -301,205 +302,6 @@ class Dashboard extends Component {
     };
   };
 
-  renderSubscriptions = () => {
```

Comment:
extracting the subscriptions menu to a separate component to be able to use the Accordion component + improve readability of Dashboard component

Reply:
Thanks for this